src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -32,10 +30,6 @@
     public static final String VALID_ADDRESS_RESIDENCE1 = "14 Amber Gardens, 439960";
     public static final String VALID_ADDRESS_RESIDENCE2 = "Block 50, Cantonment Rd";
 
-    public static final String VALID_BOOKING_DETAILS = "4 Adults";
-    //public static final List<Booking> VALID_BOOKING_LIST1 = new BookingList()
-    //        .add(new Booking(VALID_NAME_AMY, VALID_PHONE_AMY, VALID_BOOKING_START1, VALID_BOOKING_END1));
-    //public static final List<Booking> VALID_BOOKING_LIST2 = "2 Teenagers";
```

Comment:
I left this commented out to be built on later when we add more tests so could we keep sorry!